--------------------Moonraker Log Start--------------------
config_file: /home/pi/klipper_config/moonraker.conf
log_file: /home/pi/klipper_logs/moonraker.log
software_version: v0.7.1-194-geda5623

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
enable_debug_logging = False
config_path = ~/klipper_config
temperature_store_size = 600
gcode_store_size = 1000

[authorization]
cors_domains = 
	*.local
	*.lan
	*://app.fluidd.xyz
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128

[history]

[octoprint_compat]

[update_manager]

[update_manager client fluidd]
type = web
repo = cadriel/fluidd
path = ~/fluidd

[update_manager KlipperScreen]
type = git_repo
path = ~/KlipperScreen
origin = https://github.com/jordanruthe/KlipperScreen.git
env = ~/.KlipperScreen-env/bin/python
requirements = scripts/KlipperScreen-requirements.txt
install_script = scripts/KlipperScreen-install.sh

#################################################################

System Info:

***cpu_info***
  cpu_count: 4
  bits: 32bit
  processor: armv7l
  cpu_desc: ARMv7 Processor rev 3 (v7l)
  serial_number: 10000000eb3c18f1
  hardware_desc: BCM2711
  model: Raspberry Pi 4 Model B Rev 1.4
  total_memory: 1864320
  memory_units: kB

***sd_info***
  manufacturer_id: 03
  manufacturer: Sandisk
  oem_id: 5344
  product_name: SS08G
  product_revision: 8.0
  serial_number: 70cc4b0b
  manufacturer_date: 3/2017
  capacity: 7.4 GiB
  total_bytes: 7948206080

***distribution***
  name: Raspbian GNU/Linux 10 (buster)
  id: raspbian
  version: 10
  version_parts: {'major': '10', 'minor': '', 'build_number': ''}
  like: debian
  codename: buster

***virtualization***
  virt_type: none
  virt_identifier: none

***available_services***
 ['klipper', 'klipper_mcu', 'KlipperScreen', 'moonraker', 'webcamd']

***service_state***
  klipper: {'active_state': 'active', 'sub_state': 'running'}
  klipper_mcu: {'active_state': 'active', 'sub_state': 'running'}
  KlipperScreen: {'active_state': 'active', 'sub_state': 'running'}
  moonraker: {'active_state': 'active', 'sub_state': 'running'}
  webcamd: {'active_state': 'active', 'sub_state': 'running'}

***network***
  eth0: {'mac_address': 'e4:5f:01:20:a5:16', 'ip_addresses': [{'family': 'ipv4', 'address': '192.168.2.97', 'is_link_local': False}, {'family': 'ipv6', 'address': 'fe80::d5f9:881d:a46:1a30', 'is_link_local': True}]}
2021-12-16 15:48:25,344 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.2.234
2021-12-16 15:48:25,963 [app.py:log_request()] - 101 GET /websocket?token=OCYC3U5H3SV74Y5CE2NCSSA6LSLCUPBD (192.168.2.234) [_TRUSTED_USER_] 5.50ms
2021-12-16 15:48:25,964 [websockets.py:add_websocket()] - New Websocket Added: 2805345072
2021-12-16 15:49:38,305 [websockets.py:remove_websocket()] - Websocket Removed: 2805345072
2021-12-16 15:49:40,390 [app.py:log_request()] - 101 GET /websocket?token=HZYRT62KBQI54BBXR4DRPJ7P5EDDXVKL (192.168.2.234) [_TRUSTED_USER_] 5.85ms
2021-12-16 15:49:40,392 [websockets.py:add_websocket()] - New Websocket Added: 2805407056
2021-12-16 15:52:45,185 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "pusheen-black.gcode") successfully finished
2021-12-16 15:53:01,223 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "pusheen-light-grey.gcode") successfully finished
2021-12-16 16:14:55,871 [app.py:log_request()] - 404 GET /machine/device_power/devices (192.168.2.97) [No User] 20.14ms
2021-12-16 16:14:55,889 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.2.97
2021-12-16 16:14:55,919 [app.py:log_request()] - 101 GET /websocket?token=EDVSYZIOCTO4SVQJ4YJ4CT2GBJAR3CWA (192.168.2.97) [_TRUSTED_USER_] 2.03ms
2021-12-16 16:14:55,920 [websockets.py:add_websocket()] - New Websocket Added: 2805161104
2021-12-16 16:14:56,127 [app.py:log_request()] - 404 GET /machine/device_power/devices (192.168.2.97) [No User] 1.68ms
2021-12-16 16:14:56,682 [file_manager.py:get_file_list()] - Updating File List <gcodes>...
2021-12-16 16:56:50,397 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.2.234
2021-12-16 17:16:50,398 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.2.97
2021-12-16 18:17:59,273 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.2.234
2021-12-16 18:17:59,686 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "pusheen-light-grey.gcode") successfully finished
2021-12-16 18:18:07,923 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "pusheen-light-grey.gcode") successfully finished
2021-12-16 19:21:50,397 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.2.234
2021-12-16 20:17:55,632 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.2.234
2021-12-16 20:17:56,155 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "hello-kitty-black.gcode") successfully finished
2021-12-16 20:17:56,172 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-black-48x48.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-black-48x48.png>
2021-12-16 20:17:56,195 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-black-300x300.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-black-300x300.png>
2021-12-16 20:18:05,117 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "hello-kitty-red.gcode") successfully finished
2021-12-16 20:18:05,196 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-red-48x48.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-red-48x48.png>
2021-12-16 20:18:05,211 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-red-300x300.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-red-300x300.png>
2021-12-16 20:18:16,713 [shell_command.py:_check_proc_success()] - Command (/home/pi/moonraker-env/bin/python /home/pi/moonraker/moonraker/components/file_manager/metadata.py -p /home/pi/gcode_files -f "hello-kitty-white.gcode") successfully finished
2021-12-16 20:18:16,729 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-white-48x48.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-white-48x48.png>
2021-12-16 20:18:16,742 [websockets.py:build_error()] - JSON-RPC Request Error: 404
Metadata not available for <.thumbs/hello-kitty-white-300x300.png>
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 224, in execute_method
    result = await method(conn, *args, **kwargs)
  File "/home/pi/moonraker/moonraker/websockets.py", line 329, in func
    ip_addr=ws.ip_addr, user=ws.current_user))
  File "/home/pi/moonraker/moonraker/components/file_manager/file_manager.py", line 229, in _handle_metadata_request
    f"Metadata not available for <{requested_file}>", 404)
utils.ServerError: Metadata not available for <.thumbs/hello-kitty-white-300x300.png>
2021-12-16 21:21:50,398 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.2.234
